ext_localconf.php

```diff
@@ -47,22 +49,6 @@
         'class' => \Bobosch\OdsOsm\Wizard\VectordrawWizard::class
     ];
 
-    // Register icons
-    $icons = [
-        'coordinate-picker-wizard' => 'ce_wiz.png',
-        'vectordraw-wizard' => 'vector.png',
-        'ods_osm' => 'osm.png'
-    ];
-    $iconRegistry = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Imaging\IconRegistry::class);
-    foreach ($icons as $identifier => $path) {
-        $iconRegistry->registerIcon(
-            $identifier,
-            \TYPO3\CMS\Core\Imaging\IconProvider\BitmapIconProvider::class,
-            ['source' => 'EXT:ods_osm/Resources/Public/Icons/' . $path]
```

Comment:
At least on 10.4, the icons are disappeared now. Is this necessary to do?

Reply:
In V11 the icons can be registered via `Configurations/Icons.php` to cleanup `ext_localconf.php`. This was one of the reasons I limited the compaitibility to V11. 

I re-added the lines to ext_localconf.php

Reply:
Ok. I'll keep the dupplication in ext_localconf.php and will add deprecation notice.